src/main/webapp/creatives/maps/dashboardScript.js

```diff
@@ -1,23 +1,24 @@
 
-import {URL} from './utils.js'
-
 google.charts.load('current', {'packages':['corechart']});
 google.charts.setOnLoadCallback(drawChart);
 
+const BASE_URL = "https://step-capstone-team33-2020.appspot.com"
```

Comment:
Why assign the project url to BASE_URL here when you created the base url constant earlier? Couldn't you import the constant and use that here as well?

Reply:
not completely sure how the url parameter thing will work that's why I left it in like that